server/repository/src/db_diesel/requisition/requisition_row.rs

```diff
@@ -91,9 +94,13 @@ pub struct RequisitionRow {
     #[diesel(column_name = type_)]
     pub r#type: RequisitionType,
     pub status: RequisitionStatus,
+    #[ts(as = "String")]
```

Comment:
NaiveDateTime does not implement TS (should check maybe there is a features flag that implements it

Reply:
https://github.com/Aleph-Alpha/ts-rs/wiki/Feature-flags#implementing-ts-for-external-types Looks like it gets mapped to string anyway...

Reply:
Thank you, i thought there would be something like this, should have checked, added to top of stack: 50594865584a3c1bfbb2262ecf5f39cb0fd09b5a, re ran the test that generates types, not changes : )